src/AngleSharp.Css/Parser/Micro/PointParser.cs

```diff
@@ -155,7 +155,12 @@ public static CssOriginValue ParseOrigin(this StringSource source)
         /// </summary>
         public static CssBackgroundSizeValue ParseSize(this StringSource source)
         {
-            if (source.IsIdentifier(CssKeywords.Cover))
+            if (source.Content.IndexOf("/") != -1 && (source.Content.Contains("cover") || source.Content.Contains("center")))
```

Comment:
Well, this also would include, e.g., `cover-center/` or `covererd/centered` etc. - things that are definitely wrong,

The source parser also has many different sub-parsers that allow you to fully use the grammar - in this case inspect `[cover|center] [/] [cover|center]`.

Reply:
@FlorianRappl - Can you explain that a bit more please? Sorry, still trying to fully understand your code base and it's patterns.

Reply:
Hm which part? The given algorithm does not work as explained - it would also match things like `cover-center/` (contains "/" the word "cover" and the word "center").

Reply:
Okay, I was referring to your comment about source parser. However, if we want the testing to be very granular then the following code would solve the issue. Just not sure if its the matching that you want fixed, or if there something in your parser comment I am not understanding?

```
if (source.Content.IndexOf("/") != -1)
{
    string pattern1 = @"\s*center\s*/*\s*cover\s*";
    string pattern2 = @"\s*cover\s*/*\s*center\s*";

    var opts = RegexOptions.Singleline | RegexOptions.IgnoreCase;

    var match1 = Regex.Match(source.Content, pattern1, opts);
    var match2 = Regex.Match(source.Content, pattern2, opts);

    if (match1.Success || match2.Success)
    {
        source.NextTo(source.Content.Length + 1);
        return new CssBackgroundSizeValue(source.Content);
    }
}

if (source.IsIdentifier(CssKeywords.Cover))


```